templates/download/server/index.html

```diff
@@ -39,7 +39,6 @@ <h2>Ubuntu Server {{ latest_release_with_point }}</h2>
           <p class="p-card__content">
             <a class="p-button--positive is-wide" href="/download/server/thank-you?version={{ latest_release_with_point }}&amp;architecture=amd64" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Download', 'eventAction' : 'Server', 'eventLabel' : '{{ latest_release }}', 'eventValue' : undefined });">Download</a></p>
             <p><small>For other versions of Ubuntu including torrents, the network installer, a list of local mirrors, and past releases <a href="/download/alternative-downloads">see our alternative downloads</a>.</small></p>
-          </p>
```

Comment:
If you have time, you could reformat this by putting the `</p>`s on their own lines - to avoid the confusion which presumably led to the extra `</p>` in the first place.

Reply:
only for you...